src/components/Card/Card.jsx

```diff
@@ -1,7 +1,13 @@
 import { Link } from 'react-router-dom'
+import { useCart } from '../../contexts/CartContext'
```

Comment:
suggestion:here you can import all custom hooks into one file then from that you can import required hooks wherever you want from single file